config/crd/bases/falcon.crowdstrike.com_falconcontainers.yaml

```diff
@@ -1877,6 +1877,14 @@ spec:
                         x-kubernetes-int-or-string: true
                     type: object
                 type: object
+              installNamespace:
+                default: falcon-sidecar
```

Comment:
Changing the default namespace to `falcon-sidecar` from `falcon-system`; does this require prior notification to customers?
(i,e) If falcon operator aborts/evicted(and pulls the latest image on restart), would it reconcile/remove the injector in `falcon-system` namespace?

Reply:
Kept default as `falcon-system` and will update the eks-fargate docs to recommend namespace change for mixed eks+fargate clusters.